dagr/dagr.py

```diff
@@ -304,6 +308,9 @@ def get_images(self, mode, mode_arg, pages):
                 except DagrException as get_error:
                     self.handle_download_error(link, get_error)
                     continue
+                except:
+                    print("Unexpected error: ", sys.exc_info()[0])
+                    continue
```

Comment:
Not a fan of exception catch-alls, are there other download failures apart from the extension?

Else basic traceback is better and provides more info for the lazy dev to fixe his errors

Reply:
Indeed, catch-all is a code smell

However, i prefer that we don't punish the user for our failure. As a programmer, I cannot guarantee I can predict all upcoming failure. I prefer that if there's an error we didn't anticipate while downloading a file, the program doesn't halt entirely and continue downloading next entry. Hopefully if only one or few downloads are failed (and reported), the user doesn't get (too) frustrated